app/src/main/res/resources.properties

```diff
@@ -0,0 +1 @@
+unqualifiedResLocale=en-US
```

Comment:
What does that line mean? Is this the default app language?

Reply:
Yes, this is the default localization. It's a required parameter.

Reply:
Got it. Thanks for the explanation!